modules/gdscript/gdscript_editor.cpp

```diff
@@ -427,7 +428,30 @@ void GDScriptLanguage::debug_get_globals(List<String> *p_globals, List<Variant>
 }
 
 String GDScriptLanguage::debug_parse_stack_level_expression(int p_level, const String &p_expression, int p_max_subitems, int p_max_depth) {
-	return "";
+    List<String> names;
+	List<Variant> values;
+	debug_get_stack_level_locals(p_level, &names, &values, p_max_subitems, p_max_depth);
+
+	Vector<String> name_vector;
+	for (auto it = names.begin(); it != names.end(); ++it) {
+		name_vector.push_back(*it);
+	}
+
+	Array value_array;
+	for (auto it = values.begin(); it != values.end(); ++it) {
+		value_array.push_back(*it);
+	}
+
+	Expression expression;
+	if (expression.parse(p_expression, name_vector) == OK) {
+        ScriptInstance *instance = debug_get_stack_level_instance(p_level);
+        if (instance) {
+            Variant return_val = expression.execute(value_array, instance->get_owner());
+            return String(return_val);
```

Comment:
I'm not familiar with the Debugger, but it might make sense to use a more useful string representation:

```suggestion
            return return_val.get_construct_string();
```